highfidelity/hifi
# WL 21493 - Update the Chat.js to indicate when a message is too long

> Update the Chat.js script from scripts/system/chat.js to indicate when a message is too long to send and prevent the user from entering text longer than the length. Currently, long messages are arbitrarily cut off (this may be a restriction on the amount of data that can be sent via messages?)
> 
> This task includes two parts:
> 
> - Investigate where the cutoff for message length is recurring and create a character restriction 
> - Update the text input to stop accepting text past the limit of what will appear in the text box so that the user is informed before sending them message what they will accurately be sending.
> Worklist https://worklist.net/21493

## Test Plan
- Run the script `scripts/system/chat.js`
- Write in a bunch of text and confirm that after 256 characters, text input is no longer permitted.
- Try to paste some text into the field too, ensuring it doesn't go over.

(Personally) I was able to quickly confirm abcdefghijklmnopqrstuvwxyz pasted 10 times resulted in 'wxyz' being cut off on the last paste equaling 256.
